tests/torchtune/modules/loss/test_kd_losses.py

```diff
@@ -17,7 +17,7 @@ def random():
 
 
 class TestForwardKLWithChunkedOutputLoss:
-    def test_forward_kl_loss(self):
+    def setup_forward_kl_loss(self, set_all_masks_zero: bool = False):
```

Comment:
nit: I would do something like "ignore_all_tokens" instead of "set_all_masks_zero" as I think it's clearer

Reply:
Sounds better, and updated.